rdmo/management/management/commands/merge_attributes.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+from django.core.management.base import BaseCommand, CommandError
+from django.utils.translation import gettext_lazy as _
+
+from rdmo.management.merge_attributes import ReplaceAttributeOnElements
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--source',
+            help='The URI of the source attribute that will be replaced by the target and will be deleted')
+        parser.add_argument(
+            '--target',
+            help='The URI of the target attribute that will be used to replaced the source')
+        parser.add_argument(
+            '--save',
+            action='store_true',
+            help='If specified, the changes will be saved and the source attribute will be deleted. If not specified, the command will do a dry run.'  # noqa: E501
+        )
+
+    def handle(self, *args, **options):
+        try:
+            ReplaceAttributeOnElements(
```

Comment:
Sorry, why is this a class? This is a fairly simple task which does not need this amount of indirection and can be coded in a procedural way. Please use **simpler** Python code. This script should be written so that people with minimal Python experience can understand it. Either rewrite the script or implement that locally.

Reply:
alright, I've refactored it now. Think that it looks much more understandable now.